web/tests/recent_topics.test.js

```diff
@@ -152,17 +181,36 @@ mock_esm("../src/unread", {
         }
         return 1;
     },
+    num_unread_for_user_ids_string() {
+        return 0;
+    },
     topic_has_any_unread_mentions: () => false,
 });
 
+mock_esm("../src/muted_users", {
+    is_user_muted(user_id) {
+        if (user_id === 5) {
+            return false;
```

Comment:
Why not use the actual functions here (from `user_topics.js`) instead of patching these? The muted topic store is pretty simple (without much dependencies to worry about). See `user_topics.test.js` for example tests where the actual `user_topics.js` code is used.